### PR TITLE
Startup And Windows CI Fixes

### PR DESCRIPTION
**Pull request by rjhancock**

### CODE REVIEW

**Comment by HoneySkull** [comment]
I pulled this and was able to build locally on Windows using Gradle and verified the resulting exe launched megamek successfully.

File: megamek/build.gradle [hunk 1]
```diff
@@ -50,7 +56,12 @@ application {
 }
 
 ext {
-    mmJvmOptions = ['-Xmx4096m', '--add-opens', 'java.base/java.util=ALL-UNNAMED', '--add-opens', 'java.base/java.util.concurrent=ALL-UNNAMED', '-Dsun.awt.disablegrab=true']
+    mmJvmOptions = [
+        '-Xmx2048m',
```
Comment (HoneySkull) [inline megamek/build.gradle:60]: Should this default to 4096m?

File: megamek/build.gradle [hunk 2]
```diff
@@ -243,21 +255,19 @@ distributions {
 createExe {
     description = 'Create Windows executable that wraps MM jar'
     mainClassName = application.mainClass
-    outfile = 'MegaMek.exe'
```
Comment (HoneySkull) [inline megamek/build.gradle:246]: This outfile assignment can be left in safely.   Was tested.

File: megamek/build.gradle (hunk 1 above)
Comment (rjhancock) [reply megamek/build.gradle:60]: No, I upped it when I shouldn't have. It should be 2G and not 4G

File: megamek/build.gradle (hunk 2 above)
Comment (rjhancock) [reply megamek/build.gradle:246]: Can be but not needed. It's default to project name plus exe.

**Comment by HammerGS** [comment]
Should we poll the community and move to 4?

**Comment by rjhancock** [comment]
4 what? Repos?

**Comment by HammerGS** [comment]
Gig of memory

**Comment by rjhancock** [comment]
Poll it. We can bump all 3 at the same time.

**Comment by kuronekochomusuke** [comment]
are 2 copies of SubmitBug.html needed?

image

**Comment by rjhancock** [comment]
No, but I can do cleanup of this stuff before the merge.

**Pull request merged**